index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My portfolio</title>
+    <!-- Google Icons --> 
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div id="root">
+        <header class="top_header">
+            <nav class="nav_bar">
+                <div class="left_side">
+                    <a href="#" class="logo">H.Monda</a>
+                </div>
+                <div class="right_side">
+                    <i class="material-icons menu_icon">menu</i>
+                </div>
+            </nav>
+            <div class="portfolio_presentation">
+                <h3 class="title">I’m Monda
+                    Glad to see you!</h3>
+                    <p class="text_port">
+                        I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                    </p>
+                    <h1>LET'S CONNECT</h1>
```

Comment:
- [ ] Kindly replace `<h1>` with `<h2>`
- [ ] Please open this link to learn more about h1 to h6 tag: [h1 to h6 tags](https://www.w3schools.com/tags/tag_hn.asp)

Reply:
- [ ] Please, kindly do the same hereby implementing the changes required by the previous reviewer because they're required in your project